makefile

```diff
@@ -124,18 +124,20 @@ debug: cxx ctr
 
 clean:
 	rm -rf $(BUILDDIR)
-	make -C $(LIBSOCKETDIR) clean
+	$(MAKE) -C $(LIBSOCKETDIR) clean
+	$(MAKE) -C src/lib/tcc clean
 
 cxx:
-	@ echo -e "\033[31;1mblah \033[32;1mblah \033[33;1mblah\033[0m"
```

Comment:
Don't do that, `echo` wouldn't interpret the escapes if you don't pass `-e`.

Reply:
it does for me for some reason, and I know I need to use `-e`, but sometimes it just echoes `-e blah blah blah`